controller/helper/process-rule.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+/**
+ * @module controller/helper/process-rule
+ */
+
+const moment = require('moment');
+
+/**
+ * A rule processing for calculating end date of patient in trial.
+ * @param {Object} ruleTotal - aggregated rule values
```

Comment:
Ok. I will do it. Forgot to update it  after changing implementation of function.